test/extended/authorization/podsecurity_admission.go

```diff
@@ -189,14 +193,302 @@ var _ = g.Describe("[sig-auth][Feature:PodSecurity][Feature:SCC]", func() {
 	})
 })
 
+var _ = g.Describe("[sig-auth][Feature:PodSecurity][Feature:LabelSyncer][Feature:ForceHistoricalOwnership]", func() {
+	defer g.GinkgoRecover()
+
+	ctx := context.Background()
+	oc := exutil.NewCLIWithoutNamespace("pod-security-managed-labels-historical")
+
+	cpc := "cluster-policy-controller"
+	psaLabelSyncer := "pod-security-admission-label-synchronization-controller"
+	ocManager := "openshift-tests"
+	labelSyncLabel := "security.openshift.io/scc.podSecurityLabelSync"
+	psaLabels := []string{
+		psapi.EnforceLevelLabel,
+		psapi.EnforceVersionLabel,
+		psapi.AuditLevelLabel,
+		psapi.AuditVersionLabel,
+		psapi.WarnLevelLabel,
+		psapi.WarnVersionLabel,
+	}
+
+	g.It("forcing PSa label historical ownership", func() {
+		for _, testCase := range []struct {
+			name                                   string
+			podSecurityLabelSync                   string
+			expectedManagerForPSaLabelsAfterCreate string
+			expectedManagerForPSaLabelsAfterUpdate string
+		}{
+			{"labelsyncer-off", "false", cpc, ocManager},
+			{"labelsyncer-on", "true", psaLabelSyncer, psaLabelSyncer},
+			{"labelsyncer-undef", "", psaLabelSyncer, ocManager},
+
+			// openshift- prefixed namespaces; special case when podSecurityLabelSync label is undefined
+			{"openshift-e2e-labelsyncer-off", "false", cpc, ocManager},
+			{"openshift-e2e-labelsyncer-on", "true", psaLabelSyncer, psaLabelSyncer},
+			{"openshift-e2e-labelsyncer-undef", "", cpc, ocManager},
+		} {
+			ns := &corev1.Namespace{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: testCase.name,
```

Comment:
use `GenerateName` here, too

Reply:
done in 01e624230e5343fa50c5a000cca0e4e8191fccf9